### PR TITLE
Feature/fsa 851 grey out translation link if none

### PR DESCRIPTION
**Pull request by EmilyHazlehurst**
Please provide a thorough description of your component and its usage with screenshots.

## Checklist before opening a PR

Before opening a PR, please make sure:

- [ ] your work is in a branch based on and synced with the develop branch.
- [ ] your scss and twig files are imported in the javascript file. Without this, Webpack won't be able to compile them.
- [ ] your newComponent is imported in the index.js file.
- [ ] your folder hierarchy follows the proper structure.
- [ ] your code follows the coding standards and naming conventions defined.
- [ ] your code has been tested as AA accessibility compliant to the WCAG 2.1 AA standard.
- [ ] your code has been tested in all the browsers listed in the GDS browser's list, including IE11.
- [ ] your code has been tested in all the different screensizes defined.
- [ ] you have run the `yarn lint` command and fixed any linting errors.
- [ ] you have updated the CHANGELOG.md file with your changes.

### CODE REVIEW

File: src/components/general/Header/header.html.twig [hunk 1]
```diff
@@ -81,23 +97,27 @@
...
                 <ul class="header__languages-list">
                   <li class="header__language">
-                    <a href="{{ english_link }}" class="header__language-link header__language-link--{{ english ? "selected" : "" }}">{{ english_label }}</a>
+                    <a href="{{ english_link }}"
+                       class="header__language-link header__language-link--{{ english ? "selected" : "" }}">{{ english_label }}</a>
                   </li>
                   <li class="header__language">
-                    <a href="{{ welsh_link }}" class="header__language-link header__language-link--{{ not english ? "selected" : "" }}">{{ welsh_label }}</a>
+                    <a href="{{ welsh_link }}"
+                       class="header__language-link header__language-link--{{ not english ? "selected" : "" }}">{{ welsh_label }}</a>
```
Comment (c-ancia) [inline src/components/general/Header/header.html.twig:109]: should we also get the not has_translation here too?

File: src/components/general/Header/header.html.twig [hunk 2]
```diff
@@ -162,10 +187,12 @@
...
       <ul class="header__languages-list">
         <li class="header__language">
-          <a href="{{ english_link }}" class="header__language-link header__language-link--{{ english ? "selected" : "" }}">{{ english_label }}</a>
+          <a href="{{ english_link }}"
+             class="header__language-link header__language-link--{{ english ? "selected" : "" }}">{{ english_label }}</a>
         </li>
         <li class="header__language">
-          <a href="{{ welsh_link }}" class="header__language-link header__language-link--{{ not english ? "selected" : "" }}">{{ welsh_label }}</a>
+          <a href="{{ welsh_link }}"
+             class="header__language-link header__language-link--{{ not english ? "selected" : "" }}">{{ welsh_label }}</a>
```
Comment (c-ancia) [inline src/components/general/Header/header.html.twig:195]: and also here? 🤔

**Review by c-ancia — CHANGES_REQUESTED** [review]
just a few questions.

Also, is it still selectable via tab key?

**Comment by EmilyHazlehurst** [comment]
@c-ancia I've made those changes.

It's not tabbable anymore, no.

**Review by c-ancia — APPROVED** [review]

**Review by c-ancia — APPROVED** [review]

**Pull request merged**